drift-transport-netty/pom.xml

```diff
@@ -133,5 +133,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Nit: we should move this above, since we put the all the test scope dependencies at the bottom (see the "for testing" comment above)